integration-tests/mongodb-panache/src/test/java/io/quarkus/it/mongodb/panache/reactive/ReactiveMongodbPanacheResourceTest.java

```diff
@@ -43,7 +42,6 @@
 
 @QuarkusTest
 @QuarkusTestResource(MongoTestResource.class)
-@Disabled("See https://github.com/quarkusio/quarkus/issues/11711")
```

Comment:
There's also `NativeMongodbPanacheResourceIT` in `integration-tests/mongodb-panache` which has been disabled for this same reason. That one needs to be re-enabled too, either in this PR or as a separate one.

Reply:
Sorry, missed that, thought it was just the one.

I'm loathed to re-trigger the full CI build, can it be in a followup if this is ok?

Reply:
> I'm loathed to re-trigger the full CI build, 

Same here.

> can it be in a followup if this is ok?

I think that's OK since if this re-enabled regular test passes, it's almost guaranteed the native one too will, given the context of this issue. So we just need to see how this one goes.